counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
```

Comment:
I only added the mpc_ prefix for the added api calls. Internally all micropayment code resides in the counterpartylib.lib.micropayments and is imported explicitly.